conanfile.py

```diff
@@ -42,8 +42,12 @@ def build(self):
                        "EXPAT_BUILD_TESTS" : "OFF",
                        "EXPAT_BUILD_TOOLS" : "OFF",
                        "CMAKE_POSITION_INDEPENDENT_CODE": "ON",
-                       "CMAKE_DEBUG_POSTFIX": "",
                        "EXPAT_MSVC_STATIC_CRT": self.options.static_crt,
+                       "CMAKE_DEBUG_POSTFIX": "",
```

Comment:
[minor] In the expat CMakeLists.txt it seems that these POSTFIX variables are only set in the `WIN32`case. I wonder if this could create problems on other platforms ...